minio_storage/storage.py

```diff
@@ -388,82 +389,116 @@ def create_minio_client_from_settings(*, minio_kwargs=None):
 
 @deconstructible
 class MinioMediaStorage(MinioStorage):
-    def __init__(self):
-        client = create_minio_client_from_settings()
-        bucket_name = get_setting("MINIO_STORAGE_MEDIA_BUCKET_NAME")
-        base_url = get_setting("MINIO_STORAGE_MEDIA_URL", None)
-        auto_create_bucket = get_setting(
-            "MINIO_STORAGE_AUTO_CREATE_MEDIA_BUCKET", False
-        )
-        auto_create_policy = get_setting(
-            "MINIO_STORAGE_AUTO_CREATE_MEDIA_POLICY", "GET_ONLY"
-        )
-
-        policy_type = Policy.get
-        if isinstance(auto_create_policy, str):
-            policy_type = Policy(auto_create_policy)
-            auto_create_policy = True
-
-        presign_urls = get_setting("MINIO_STORAGE_MEDIA_USE_PRESIGNED", False)
-        backup_format = get_setting("MINIO_STORAGE_MEDIA_BACKUP_FORMAT", False)
-        backup_bucket = get_setting("MINIO_STORAGE_MEDIA_BACKUP_BUCKET", False)
```

Comment:
Note, the previous default value of `False` violates the allowed type of `T.Optional[str]` for `MinioStorage.backup_format` and `MinioStorage.backup_bucket`. The updated default is now `None`, which has the same behavior but the correct type.